src/mlpack/methods/ann/layer/convolution.hpp

```diff
@@ -324,6 +278,9 @@ class Convolution
   //! Locally-stored transformed gradient parameter.
   arma::cube gradientTemp;
 
+  //! Locally-stored padding layer.
+  LayerTypes<> padding;
```

Comment:
I think this should be a pointer.

Reply:
yes it is already a pointer.

Reply:
Okay, I didn't know about it. Thanks

Reply:
You are right, this is already a pointer, but II'm wondering, since we already know this would be a Padding layer, we could just use the ``Padding`` type here.